server/cli/src/lib.rs

```diff
@@ -5,3 +5,9 @@ mod report_utils;
 pub use report_utils::*;
 mod graphql;
 pub use graphql::*;
+
+mod helpers;
+pub use helpers::*;
+
+mod plugins;
+pub use plugins::*;
```

Comment:
I had issues with imports, since plugins was being used directly in CLI, and once i started sharing the helper method there was some warnings ¯_(ツ)_/¯